time stamp
uint8[]  id_list

uint32[] temperature_limit__degC
uint32[] max_voltage_limit__V
uint32[] min_voltage_limit__V
uint32[] pwm_limit__degree
uint32[] current_limit__mA
uint32[] acceleration_limit__deg_ss
uint32[] velocity_limit__deg_s
uint32[] max_position_limit__deg
uint32[] min_position_limit__deg
